Documentation/advanced-configuration.md

```diff
@@ -26,6 +27,78 @@ The Kubernetes based examples assume Rook OSD pods are in the `rook-ceph` namesp
 If you run them in a different namespace, modify `kubectl -n rook-ceph [...]` to fit
 your situation.
 
+## Use custom Ceph user and secret for mounting
```

Comment:
It would be helpful to have a link from the filesystem.md to this topic so it is more discoverable